package.json

```diff
@@ -40,16 +40,19 @@
     "husky": "^8.0.2",
     "jimp": "^0.22.4",
     "js-yaml": "^4.1.0",
+    "libsodium-wrappers": "^0.7.11",
```

Comment:
I presume this is for salting given the `sodium` term

Reply:
yeah like that lol